data-otservbr-global/npc/battlemart.lua

```diff
@@ -1,4 +1,4 @@
-local internalNpcName = "Battlemart"
+local internalNpcName = "Imbuement Assistant"
```

Comment:
Melhor manter o nome original, todos vão saber para que serve o npc quando conversar com ele ```suggestion
local internalNpcName = "Battlemart"
```